domains/libraries/EnergyOrigin.TokenValidation/configuration.yaml

```diff
@@ -1 +1 @@
-version: 3.6.8
```

Comment:
We don't use nuget anymore. So we shouldn't have to bump these anymore.

Reply:
Fixed: https://github.com/Energinet-DataHub/energy-origin/pull/1492/commits/1e019e674f303254bfcb614614c8c7c494b18e35